### ISSUE TITLE
Consider OG title and OG description changes

### ISSUE THREAD

**Issue by iskrisis** [diesdasdigital/kirby-meta-knight#6]
Is there situation where 
`Title / OG Title` and `Description / OG Description`
are different? Or are there different character limits?

Maybe they should be merged so editors don't have to fill everything twice.

**Comment by jonathanmuth** [comment]
Good point! When we first started developing the plugin we had one description and one title field that would automatically be used for OG Title and OG description.

Then two of our clients requested more flexibility when sharing content on Facebook but not at the expense of altered page titles and descriptions that would show up in their search listings. That's when I decided to add the dedicated OG Title and OG description fields in the blueprint.

I agree that it creates a lot of redundant work for editors on sites that don't require this flexibility. I'll note it as a feature request for version 1.1. Maybe we can create a "full-auto" version of the blueprint + snippets that keep duplicate fields to a minimum and a "full-flexibility" version that continues to allow for manual entry of values into each field.
